deploifai/dataset/pull.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+
+import click
+
+from deploifai.clouds.utilities.data_storage.handler import DataStorageHandlerEmptyFilesException, \
+    DataStorageHandlerTargetNotFoundException
+from deploifai.context import DeploifaiContextObj, pass_deploifai_context_obj, is_authenticated, dataset_found
+from deploifai.core.data_storage import DataStorage
+
+
+@click.command()
+@click.argument("target", required=False)
+@pass_deploifai_context_obj
+@is_authenticated
+@dataset_found
+def pull(context: DeploifaiContextObj, target: str = None):
+    """
+    Download files from the cloud to local.
+    """
+
+    api = context.api
+    dataset_id = context.dataset_config["DATASET"]["id"]
+
+    data = context.api.get_data_storage_info(dataset_id)
+    if len(data) == 0:
```

Comment:
`len(data)` doesn't seem right, `get_data_storage_info` should just return one data storage object, isn't it?

Reply:
Yes
It should be `if data is None instead`.
Has been refactored